### PR TITLE
Members extension

### PR DESCRIPTION
**Pull request by JoelForamitti**
This PR introduces the members extension to register and manage member data.

Should be merged after #18 and #19.

The test-app now initializes with some test users `test_superuser_1`, `test_member_1`, and `test_member_2` (the password is `test` for all of them)

There are two menuitems for the members extension, one for users and one for admins, with two respective microfrontends. There is no webcomponent yet for the microfrontend.

There are the following two new APIs:
- Users can register as a member and read and update their data via `api/members/v1/me`
- Admins can manage member data via `api/members/v1/members`

There are further the following new API features:
- Filtering `{URL}?{attr}={value}` or  `{URL}?{attr}__gt={value}` (greater than), see [django docs](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#field-lookups) for all possible lookups (gt, gte, contains, etc.)
- Sorting `{URL}?ordering={attr}` or `{URL}?ordering=-{attr}` (reverse, note the minus sign before attr)
- Pagination `{URL}?limit={limit}&offset={offset}`

### CODE REVIEW

File: collectivo-test-app/test_extension/apps.py [hunk 1]
```diff
@@ -18,15 +18,31 @@ def post_migrate_callback(sender, **kwargs):
     register_microfrontend(
         name=sender.name+'_modules',
         extension=sender.name,
-        path='http://localhost/static/test_extension/remoteEntry.js',
-        method='modules'
+        path='http://localhost:8000/static/test_extension/remoteEntry.js',
```
Comment (nanu-c) [inline collectivo-test-app/test_extension/apps.py:21]: this should be collectivo.local:8000

File: collectivo-test-app/test_extension/apps.py [hunk 2]
```diff
@@ -18,15 +18,31 @@ def post_migrate_callback(sender, **kwargs):
...
+        type='modules'
     )
 
     register_microfrontend(
         name=sender.name+'_iframe',
         extension=sender.name,
-        path='http://localhost/test_extension/',
-        method='iframe'
+        path='http://localhost:8000/test_extension/',
```
Comment (nanu-c) [inline collectivo-test-app/test_extension/apps.py:28]: same here

File: collectivo/members/apps.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    register_menuitem(
+        name='members_user_menu_item',
+        label='My membership',
+        extension=name,
+        menu='main_menu',
+        microfrontend='members_user_ui'
+    )
+
+    register_menuitem(
+        name='members_admin_menu_item',
```
Comment (nanu-c) [inline collectivo/members/apps.py:48]: shouldn't we add here a member role like `role='members_admin'`? And then in the menu view filter it, that the ui knows, when to show the menu item and when not?

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: collectivo/members/apps.py (hunk 3 above)
Comment (JoelForamitti) [reply collectivo/members/apps.py:48]: yes, i wanted to work on this in my next PR :)

**Pull request merged**